### PR TITLE
Add a warning to Searchfox when the metric is expired

### PR DESCRIPTION
**Pull request by Iinh**
### Pull Request checklist

- [x] The pull request has a descriptive title (and a reference to an issue it
      fixes, if applicable)
- [x] All tests and linter checks are passing
- [x] The pull request is free of merge conflicts

### CODE REVIEW

File: src/pages/MetricDetail.svelte [hunk 1]
```diff
@@ -219,6 +220,11 @@
           <HelpHoverable
             content={helpText.searchFox.text}
             link={helpText.searchFox.link} />
+          {#if isExpired(metric.expires)}
+            <HelpHoverable content={'This metric has expired.'}>
```
Comment (wlach) [inline src/pages/MetricDetail.svelte:224]:
```suggestion
            <HelpHoverable content={'This metric has expired and will likely not appear in Searchfox.'}>
```

File: src/pages/MetricDetail.svelte [hunk 2]
```diff
@@ -219,6 +220,11 @@
...
+              <WarningIcon width={'15px'} height={'15px'} />
```
Comment (wlach) [inline src/pages/MetricDetail.svelte:225]: It would be nice to not hardcode this here, but maybe that's a problem to solve another time.

**Review by wlach — APPROVED** [review]
Approve with one suggestion. Thanks @Iinh!

File: src/pages/MetricDetail.svelte (hunk 2 above)
Comment (Iinh) [reply src/pages/MetricDetail.svelte:225]: It was not possible to pass these values down as props in `HelpHoverable`. I hope that this use case is rare enough that hard coding is okay, for now.

**Pull request merged**